dll/win32/syssetup/wizard.c

```diff
@@ -2831,6 +2831,53 @@ ProcessUnattendSection(
     }
 
     RegCloseKey(hKey);
+
+    if (SetupFindFirstLineW(pSetupData->hSetupInf,
+        L"Env",
+        NULL,
+        &InfContext))
+    {
+        if (RegCreateKeyExW(
```

Comment:
That registry key should already exist; if it doesn't then it should be an error.

Reply:
The only reason why I would see this fail is when the key cannot be opened for some reason?